.github/actions/ci-foundry/action.yml

```diff
@@ -56,7 +56,7 @@ runs:
       uses: Rubilmax/foundry-gas-diff@v3.9
       with:
         report: ${{ inputs.protocol }}.${{ inputs.network }}.gasreport.ansi
-        ignore: test-foundry/**/*
+        ignore: test/**/*,lib/**/*
```

Comment:
it will remove every gas diff no?

Reply:
It will remove every gas diff of contracts declared under the `test` or the `lib` directory
And it's the expected behavior because in this repository, we only want the gas diff of contracts defined in this repository (which corresponds to the `SupplyVault` et al)